operations/rates.md

```diff
@@ -576,6 +577,126 @@ Note that prices are defined daily, so when the server receives the UTC interval
 {}
 ```
 
+## Set rates
```

Comment:
Will this operation eventually replace `Add rates` or does it serve a different purpose? if we are going to have both, we need to say what the difference is, i.e. when would you use this operation compared to the other operation?

Reply:
This one has "upsert" behaviour - it can add new entities or update the existing ones. "Add" operation is just for creation of new resources. We usually have "add" and "update", but this one was geared towards Choice's requirements specifically.